fletcher/base.py

```diff
@@ -313,90 +280,6 @@ def _reduce(self, name, skipna=True, **kwargs):
             )
         )
 
-    def __setitem__(self, key, value):
```

Comment:
this's not required by pandas ! it looks strange to want to modify elements of immutable data structure.